charts/IndicatorDropdown.tsx

```diff
@@ -69,9 +35,12 @@ export class IndicatorDropdown extends React.Component<IndicatorDropdownProps> {
     }
 
     @bind async loadOptions(query: string): Promise<Indicator[]> {
-        return await this.context.indicatorStore.search({
+        const entries = await this.context.store.indicators.search({
             query
         })
+        return entries
+            .filter(entry => entry.entity) // remove entities that haven't loaded
```

Comment:
I'm confused by this too. How are we going to get only partial loading? We're loading a single static JSON file that's a list of indicators. Why do we have to filter it for missing things?

Reply:
Yeah not really necessary since `search()` returns loaded entities only. 

I added the filter because Typescript complained that `entry.entity` can technically be `undefined`.

Reply:
Removed it now since `search()` probably won't ever return entities that aren't loaded.